src/OV/OpenVarioMenu.cpp

```diff
@@ -304,8 +304,13 @@ class MainMenuWidget final
   void Show(const PixelRect &rc) noexcept override {
     RowFormWidget::Show(rc);
 
-    if (remaining_seconds > 0)
+    if (remaining_seconds > 0) {
       ScheduleTimer();
+    }
+    else {
+	  HideRow(Controls::TIMER);
+      StartXCSoar();
+	}
```

Comment:
Indentation broken, and I don't understand the commit message.

Reply:
fixed